dbt_project.yml

```diff
@@ -1,7 +1,8 @@
 
 name: 'linkedin'
-version: '1.0.0'
+version: '0.2.0'
 config-version: 2
+require-dbt-version: [">=0.18.0", "<0.19.0"]
 
 source-paths: ["models"]
```

Comment:
similar to my comment in the source package, please remove this block of default code. Thanks!